lepton/const.go

```diff
@@ -21,26 +21,34 @@ const manifest string = `(
 const Version = "0.2"
 const OpsReleaseUrl = "https://storage.googleapis.com/cli/%v/ops"
 
+// .ops directory
+const OpsDir string = ".ops"
+
 // boot loader
 const BootImg string = ".staging/boot.img"
 
 // kernel
 const KernelImg string = ".staging/stage3.img"
 
 // kernel + ELF image
-const mergedImg string = ".staging/tempimage"
+const mergedImg string = "tempimage"
 
 // final bootable image
 const FinalImg string = "image"
 
 const Mkfs string = ".staging/mkfs"
 
+// file which describes contents of tmp data
+const TMPContentsFile string = "contents.json"
+const TMPStagingDirectoryPrefix string = "staging_"
+
 const ReleaseBaseUrl string = "https://storage.googleapis.com/nanos/release/%v"
 const DevBaseUrl string = "https://storage.googleapis.com/nanos/%v"
 
 const PackageBaseURL string = "https://storage.googleapis.com/packagehub/%v"
 const PackageManifestURL string = "https://storage.googleapis.com/packagehub/manifest.json"
 const PackageManifestFileName string = "manifest.json"
+const PackageRootPath string = "sysroot"
```

Comment:
This should not be a global const if it's used only in one function.

Reply:
Hm.. I thought it's better to move "sysroot" to const instead of hardcoding. If you think it's not needed, I'll remove it

Reply:
I am fine if you want to keep it a const but then don't export it.(ie uppercase P)

Reply:
ah, okay, I'll make it not exported